src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue

```diff
@@ -472,6 +472,7 @@ export class TreeBuilderToolbar extends mixins(Routes, Permissions, Lang) {
       type,
       ui_metadata: {
         canvas_coordinates: computeBlockUiData(this.activeBlock),
+        should_auto_update_name: true,
```

Comment:
- [ ] make this optional as in flow-runner recommendation https://github.com/FLOIP/flow-runner/pull/78/files#diff-8e0400228dd698ac38c3bdcccec67c81d0c45614354d8cf60f4e6af3dd863aebR52